Follow up on open issue on the same topic: instalation in AzureML guide

elikling opened the pull request:
One MarkDown file describing the neuances of installing within AzureML

amit-sharma commented:
thanks @elikling I've modified your file to add the pip and conda instructions and created a general install file for dowhy. Hope this works. I also made slight edits to your proposal. 

Let me know if this sounds good, and we can go ahead and merge this.

elikling commented:
I comitted a minor change: deleted a supreflous space and changed "an python" to "a python". Do I need to re-pull-request this or the comit is sufficiant for the review?

amit-sharma commented:
Ah, yes. That is a great point. I was thinking that we could merge this single file with Eli's contribution, while I work on a separate PR that includes the full skeleton.

I like your idea. Let me raise that PR separately, and perhaps we can merge this after that PR is merged.

elikling commented:
I think that next time I should create a new branch in my fork and try and pull it into the parent - or perhaps we shoudl close this pull so I can open a new one?

amit-sharma commented:
@elikling it is usually better to have a single goal for each PR. With the new commits, there are two additions, 1) one for installation docs, 2) the other for FDR control. 
It will be best to separate them. Can you try closing this PR and then 
1) issuing a new PR with only the docs/installation instructions? Since we have reviewed it once, it should be quick to approve and merge.
2) then you can raise a separate PR for the FDR control, which can be then reviewed. 

What do you think?

elikling commented:
I will try and figure out how to do that

I have to use a fork as I cannot branch in yours. I guess I should be able to branch the fork?

Lets see what happens - I will be happy to learn how to do this more effectively.

With Regard,
Eli Y. Kling
Nothing is certain but the unforeseen
Acer Corner | 10 Manor Road | Wendover | HP22 6HQ |Buckinghamshire | United Kingdom
Tel: +44 (0)1296 696 017 | Mob +44 (0)795 601 60 22 | www.acercorner.com<http://www.acercroner.com/>
***@***.******@***.***>
Please Link-In with me: Uk.linkedin.com/in/elikling/<http://uk.linkedin.com/in/elikling/>

From: Amit Sharma ***@***.***>
Sent: 05 July 2022 10:04
To: py-why/dowhy ***@***.***>
Cc: Eli Y. Kling ***@***.***>; Mention ***@***.***>
Subject: Re: [py-why/dowhy] Follow up on open issue on the same topic: instalation in AzureML guide (PR #473)


@elikling<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Felikling&data=05%7C01%7C%7Cf7da401cd0584e4dba8808da5e654927%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637926086351935440%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kpXUVXhg1KaGLWFsh6uGUY17om7F3%2BAqqtU59Q1SgqY%3D&reserved=0> it is usually better to have a single goal for each PR. With the new commits, there are two additions, 1) one for installation docs, 2) the other for FDR control.
It will be best to separate them. Can you try closing this PR and then

  1.  issuing a new PR with only the docs/installation instructions? Since we have reviewed it once, it should be quick to approve and merge.
  2.  then you can raise a separate PR for the FDR control, which can be then reviewed.

What do you think?

-
Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fpy-why%2Fdowhy%2Fpull%2F473%23issuecomment-1174809058&data=05%7C01%7C%7Cf7da401cd0584e4dba8808da5e654927%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637926086351935440%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kLDUjQVwMbhwqFVV070SIXFLXmVTlwGYd9Izq8np3Og%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FACBI73WI3SMJZXW4RL4VYNTVSP27TANCNFSM5ZB3R37A&data=05%7C01%7C%7Cf7da401cd0584e4dba8808da5e654927%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637926086351935440%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=a%2Fj2VAmMDafiAkco4lsGzkD1G5ExUHt9ppJRu%2F2e53A%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

elikling commented:
I will be closing this pull reuest at Amit's guidance so as to keep them tidy

amit-sharma commented:
There are many ways, but it might be easiest if you can create a single fork, create a PR for the docs, and then we can merge it to dowhy. Then you can create a fresh fork off the dowhy master branch (deleting your old one if required), and then submit the next PR.

elikling commented:
OK –

I deleted the “offending” files 😉 and closed the PR does that work from your side.

With Regard,
Eli Y. Kling
Nothing is certain but the unforeseen
Acer Corner | 10 Manor Road | Wendover | HP22 6HQ |Buckinghamshire | United Kingdom
Tel: +44 (0)1296 696 017 | Mob +44 (0)795 601 60 22 | www.acercorner.com<http://www.acercroner.com/>
***@***.******@***.***>
Please Link-In with me: Uk.linkedin.com/in/elikling/<http://uk.linkedin.com/in/elikling/>

From: Amit Sharma ***@***.***>
Sent: 05 July 2022 11:42
To: py-why/dowhy ***@***.***>
Cc: Eli Y. Kling ***@***.***>; State change ***@***.***>
Subject: Re: [py-why/dowhy] Follow up on open issue on the same topic: instalation in AzureML guide (PR #473)


There are many ways, but it might be easiest if you can create a single fork, create a PR for the docs, and then we can merge it to dowhy. Then you can create a fresh fork off the dowhy master branch (deleting your old one if required), and then submit the next PR.

—
Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fpy-why%2Fdowhy%2Fpull%2F473%23issuecomment-1174910235&data=05%7C01%7C%7C468b1d105e0640ebaad608da5e72f310%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637926145042889289%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=pWZh8dSPyZNOyQNLVo%2FBIjM97NXndPCGqdCKHkaca%2FU%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FACBI73RFQPROH7IJAK7ZYOLVSQGOLANCNFSM5ZB3R37A&data=05%7C01%7C%7C468b1d105e0640ebaad608da5e72f310%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637926145042889289%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=bqh8nsfjPp4F4Kr0eQHFPQh8BBrssIxE6U9VfYd1ixc%3D&reserved=0>.
You are receiving this because you modified the open/close state.Message ID: ***@***.******@***.***>>

elikling commented:
Could you let me know if you are done with this pull request so as I can re-sync this forc and work on another pull request?

amit-sharma reviewed (approved):
Adds installation details for Azure.
